project/src/main/scala/org/broadinstitute/clio/sbt/Dependencies.scala

```diff
@@ -7,31 +7,32 @@ import sbt._
   */
 object Dependencies {
   // The version numbers of each main and test dependency.
-  private val AkkaHttpCirceVersion = "1.19.0"
-  private val AkkaHttpVersion = "10.0.11"
-  private val AlpakkaVersion = "0.17"
+  private val AkkaVersion = "2.5.11"
```

Comment:
no longer alpha sorted

Reply:
I strip `Version` in my head before sorting